public/employers/ruths-chris.yml

```diff
@@ -0,0 +1,51 @@
+name: Ruth's Chris Steak House
+aliases:
+  - Ruth's Hospitality Group
+shortName: Ruth's Chris
+wiki: Ruth's Chris Steak House
+image: ruths-chris.svg
+officialWebsite: https://ruthschris.net/about-us/ruths-story/
+industries:
+  - restaurant
+location:
+  city: Winter Park
+  state: FL
+  country: us
+  international: true
+  wiki: Winter Park, Florida
+employeesBefore:
+  lowerBound: 5000
+  type: range
+  upperBound: 0
+  year: 2020
+  yearQuarter: Q1
+summary: Ruth's Chris has cut salaries&mdash;including those of executives&mdash;and closed locations to cut costs. However, despite that and significant cash reserves, on top of government loans, it has still furloughed a significant number of employees.
+citations:
+  - summary: Ruth's Chris has closed 23 restaurant locations (of about 135) temporarily, saying that takeout options were not viable at those locations. Some employees have been redeployed to locations which remain open for takeout or delivery.
+    positivity: 1
+    type: publication
+    sources:
+      - source: Nation's Restaurant News
+        link: https://web.archive.org/web/20200401100012/https://www.nrn.com/finance/ruths-chris-steak-house-closes-23-restaurants-wake-coronavirus-pandemic
+        date: 2020-03-30T18:20:59Z
+  - summary: Salaried employees, including executives, at Ruth's Chris have taken pay cuts. Details on how significant the cuts were have not been released.
+    positivity: 0
+    type: publication
+    sources:
+      - source: Nation's Restaurant News
+        link: https://web.archive.org/web/20200401100012/https://www.nrn.com/finance/ruths-chris-steak-house-closes-23-restaurants-wake-coronavirus-pandemic
+        date: 2020-03-30T18:20:59Z
+  - summary: Ruth's Chris has furloughed a "significant number" of workers, though extensive details have not been provided. This has happened despite their application for a PPP loan which was created with the intent to pay employees rather than furlough them.
+    positivity: -1
+    type: publication
+    sources:
+      - source: Nation's Restaurant News
+        link: https://web.archive.org/web/20200401100012/https://www.nrn.com/finance/ruths-chris-steak-house-closes-23-restaurants-wake-coronavirus-pandemic
+        date: 2020-03-30T18:20:59Z
+  - summary: Despite closing a number of locations, an annual revenue of almost $470 million USD, and cash reserves of around $86 million USD, Ruth's Chris has, in less than a week, burned through their entire $20 million USD government loan.
```

Comment:
They might have some debts to pay, but I'm not sure what their financial situation is.

Reply:
I'm not sure either, but the PPP loan is intended to pay down employees, and if they're actually sitting on $86m cash and aren't even trying to excuse it... seems a little odd to me.